webapp/passport/js/d3.v3.js

```diff
@@ -0,0 +1,9466 @@
+!function() {
```

Comment:
Do we need another copy of d3? Can we use the one we ship with normally? Or is it a different version?

Reply:
using the existing version.